.travis.yml

```diff
@@ -24,10 +24,12 @@ cache:
 install:
     - echo yes | ${ANDROID_HOME}/tools/bin/sdkmanager "platforms;android-28"
 
+before_script:
+  - bash scripts/prep-key.sh
+
 script:
-  - 'if [ "$TRAVIS_BRANCH" != "apk" ]; then ./gradlew build --stacktrace; fi'
```

Comment:
This was added to prevent unwanted build for pushes to apk branch. Revert

Reply:
@CloudyPadmal Its fine. there need not be a check for apk branch as apk branch does not has .tavis.yml file. There wont be a travis run for this branch which is correct.